auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -789,19 +789,20 @@ public String getIdentityProviderMetadataXml(String accountId) {
         return resourceRoleAssignments;
     }
 
-    public void assignResourceRole(
-            String assigneeCrn, String resourceCrn, String resourceRoleCrn, Optional<String> requestId) {
+    public void assignResourceRole(String assigneeCrn, String resourceCrn, String resourceRoleCrn, Optional<String> requestId) {
         UmsClient client = makeClient(channelWrapper.getChannel());
-        LOGGER.info("Assigning {} role for resource {} to assignee {}", resourceRoleCrn, resourceCrn, assigneeCrn);
+        LOGGER.info("Assigning '{}' role at resource '{}' for assignee '{}'...", resourceRoleCrn, resourceCrn, assigneeCrn);
```

Comment:
I think `for assignee` is not grammatically correct. In case of person, we are assigning something to someone, not "something for someone".